integration-tests/package-lock.json

```diff
@@ -1,9150 +1,8 @@
 {
 	"name": "integration-tests",
 	"version": "13.0.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```

Comment:
what npm version are you using? I think we'd like to make sure the lockfileVersions are consistently `2`

Reply:
running 6.14.17

Reply:
which npm version should i be running?

Reply:
i have to run node 14 to make lerna commands happy

Reply:
Roxane and I both run npm version 8.1.0

Reply:
the lock files should auto adjust to version 2 if you use version 8 and above I believe